service/src/main/kotlin/app/cash/backfila/ui/pages/BackfillShowAction.kt

```diff
@@ -323,9 +329,9 @@ class BackfillShowAction @Inject constructor(
 
         // Smart auto-reload section for Events (preserves expanded state)
         AutoReload(frameId = "backfill-$id-events") {
-          Card {
+          FullWidthCard {
             // Events
-            div {
+            div("mx-auto max-w-7xl sm:px-6 lg:px-8") {
```

Comment:
Don't we want something like `max-w-max` ? We want the main window to take up the remainder but with some min size.